src/github.com/matrix-org/dendrite/clientapi/routing/register.go

```diff
@@ -17,20 +17,11 @@ package routing
 
```

Comment:
why are we making `register.go` (and `user_interactive_auth_flow.go`) executable? please don't do that.

Reply:
are you talking about 644 permissions?

Reply:
Sorry, I didn't notice that.I dont remember changing it. :/
 Anyways, I will fix it.

Reply:
looks better now, thanks!